java/email-contact-form/src/Utils.java

```diff
@@ -0,0 +1,140 @@
+package io.openruntimes.java.src;
+
+import java.io.File;
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Map;
+import jakarta.mail.MessagingException;
+import jakarta.mail.Session;
+import jakarta.mail.Transport;
+import jakarta.mail.internet.InternetAddress;
+import jakarta.mail.internet.MimeMessage;
+import jakarta.mail.Message;
+import java.util.List;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.util.Properties;
+import java.io.UnsupportedEncodingException;
+import java.net.URLDecoder;
+
+
+public class Utils {
+
+    static String INDEX_HTML = "<!doctype html><html lang=\"en\"><head><meta charset=\"UTF-8\" /><meta http-equiv=\"X-UA-Compatible\" content=\"IE\" /><meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\" /><title>Email Contact Form</title><link rel=\"stylesheet\" href=\"https://unpkg.com/@appwrite.io/pink\" /><link rel=\"stylesheet\" href=\"https://unpkg.com/@appwrite.io/pink-icons\" /></head><body><main class=\"main-content\"><div class=\"top-cover u-padding-block-end-56\"><div class=\"container\"><div class=\"u-flex u-gap-16 u-flex-justify-center u-margin-block-start-16\"><h1 class=\"heading-level-1\">Contact</h1><code class=\"u-un-break-text\"></code></div><p class=\"body-text-1 u-normal u-margin-block-start-16\" style=\"max-width: 50rem\">Fill the form below to send us a message.</p><form class=\"u-flex-vertical u-gap-16 u-margin-block-start-16\" style=\"max-width: 50rem\" method=\"post\"><input type=\"email\" name=\"email\" placeholder=\"Email\" required /><textarea name=\"message\" placeholder=\"Your Message\" required></textarea><button class=\"button\" type=\"submit\">Submit</button></form></div></div></main></body></html>";
```

Comment:
We're going to hold off merging this PR until we have a better solution for this on Open Runtimes